1_fetch/src/fetch_sites.R

```diff
@@ -71,7 +71,7 @@ fetch_rapid_dep_sites <- function(ind_file, event) {
     rdgs <- data.frame()
   } else {
     # fetch the rapid deployment gages (RDGs) from the STN web services
-    url <- sprintf('https://stn.wim.usgs.gov/STNServices/Instruments/FilteredInstruments.json?Event=%d&EventType=&EventStatus=&States=&County=&CurrentStatus=&CollectionCondition=&SensorType=5&DeploymentType=', event)
+    url <- sprintf('https://stn.wim.usgs.gov/STNServices/Instruments/FilteredInstruments.json?Event=%d&EventType=&EventStatus=&States=&County=&CurrentStatus=&CollectionCondition=&SensorType=1&DeploymentType=1,2', event)
```

Comment:
I don't think we want to commit this change - this is what allowed us to show sensors other than the "rapid deployment gages", but I think we should make that decision to expand beyond RDGs per event.